blog/2019-08-ng-deploy/README.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "All you need to know about `ng deploy`"
+author: Johannes Hoppe
+mail: johannes.hoppe@haushoppe-its.de
+published: 2019-08-23
+keywords:
+  - Angular
+  - Angular CLI
+  - deployment
+  - ng deploy
+  - ng-deploy
+language: en
+thumbnail: space-shuttle.jpg
+hidden: true
+---
+
+With version 8.3 of the Angular CLI a new command has been released which has the potential to be a game changer.
+
+<hr>
+
+Table of contents:
+
+TODO
+
+# Command Line Call
+
+Let's take we look at the semantics. How is a deploy builder called?
+
+Without any further arguments, the CLI will try to execute the deploy builder for the default project:
+
+```bash
+ng d
+```
+
+Which is the same as:
+
+```bash
+ng deploy
+```
+
+As you might know, you can manage several Angular Apps (projects) in one workspace. The CLI accepts an optional project name, as specified in the `angular.json` configuration file.
+
+```bash
+ng run [PROJECT_NAME]:deploy

```

Comment:
I'd rather not show `ng run` here but instead, in order to stick with "one command to rule them all":

```
ng deploy --project=[PROJECT_NAME]
```